src/components/flat-table/flat-table.stories.mdx

```diff
@@ -865,6 +865,194 @@ Balance performance concerns with the convenience of the user being able to see
   </Story>
 </Preview>
 
+### Paginated with sticky header
```

Comment:
Please could you add a paragraph here that explains the ActionPopover behaviour in row 5. As it is, you'd have to really look for it to see that this is possible.